src/sonic-yang-models/tests/yang_model_pytests/test_crm.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+
+thresholds = [
+    'free', 'FREE',
+    'used', 'USED',
+    'percentage', 'PERCENTAGE'
+]
+
+
+@pytest.fixture(params=thresholds)
+def threshold(request):
+    return request.param
+
+
+resources = [
+    'ipv4_route',
+    'ipv6_route',
+    'ipv4_nexthop',
+    'ipv6_nexthop',
+    'ipv4_neighbor',
+    'ipv6_neighbor',
+    'nexthop_group_member',
+    'nexthop_group',
+    'fdb_entry',
+    'mpls_nexthop',
+    'srv6_nexthop'
```

Comment:
original test has acl_counter and dnat_entry, but this new test does not have them?

Reply:
Good catch, thank you!